test/controllers/home_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
```

Comment:
delete this test folder completely

Reply:
@mujeeb-arshad , do I need to delete the test folder or home_controller folder

Reply:
test folder

Reply:
Done